doc/HOWTO/keys.txt

```diff
@@ -68,5 +68,42 @@ you don't want your key to be protected by a password, remove the flag
     type in the password every time the server needs to access
     the key.
 
+
+4. To generate an EC key
+
+An EC key can be used both for key agreement (ECDH) and signing (ECDSA).
+
+Generating a key for ECC is similar to generating a DSA key. These are
+two-step processes. First, you have to generate EC parameters from which
+the key will be generated:
+
+  openssl ecparam -name prime256v1 -out prime256v1.pem
+
+The prime256v1 (a.k.a NIST P-256), which stands for 'X9.62/SECG curve over
+a 256 bit prime field', is the name of an elliptic curve which generates the
+parameters. You can use the following command to list all supported curves
+in OpenSSL:
```

Comment:
I'd say "in OpenSSL" is totally redundant... Or it actually makes statement formally incorrect, because referred command does not list curves in OpenSSL, but those available in given compilation. I mean "OpenSSL supports curve X" doesn't necessarily mean that it appears in custom-build output...